### DISCUSSION TITLE
Integrat new Spotify Password

### DISCUSSION THREAD

**Discussion by ArneWindler** [MiczFlor/RPi-Jukebox-RFID#2422] [Q&A]
Hello, 
I had to change my Spotify password and since then the Spotify integration on my Phonibox no longer works properly. I changed the password as well as the client_id and client_secret in the mopidy.conf file and also reconnected to Spotify with Iris. In Iris I can also see some of my Spotify playlists and also the albums that I have added to the Phonibox. When I start a track, it plays for the first 4 seconds and then starts again, but I can't hear anything. My question is therefore, where and how do I have to change the Spotify password so that the integration works again?
I appreciate any advice!

**Comment by AlvinSchiller** [comment]
Hi, Spotify has changed there API, so the login with username and password don't seem to work anymore.
This is not fixed in mopidy yet:
https://github.com/mopidy/mopidy-spotify

There are some workarounds that might work in your case, but I didn't try any myself.

See also https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2418#issuecomment-2297264187

**Comment by s-martin** [comment]
Please follow up in #2429

**Reply by seschu** [reply]
I would like to know if this is a solution https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2429#issuecomment-2409105287

**Comment by s-martin** [comment]
Fixed in #2435 

will be part of 2.8
